src/scenes/Species/DeleteSpeciesModal.tsx

```diff
@@ -46,60 +44,30 @@ export default function DeleteSpeciesDialog(props: DeleteSpeciesDialogProps): JS
 
   useEffect(() => {
     if (inUseSpecies) {
-      const forDeletion = speciesToDelete
-        .filter((species) => !inUseSpecies[species.id.toString()])
-        .map((species) => species.id);
-      const avoidDeletion = speciesToDelete
-        .filter((species) => inUseSpecies[species.id.toString()])
-        .map((species) => species.id);
-      setToDelete(forDeletion);
-      setCannotDelete(avoidDeletion);
+      if (inUseSpecies[speciesToDelete.id.toString()]) {
```

Comment:
Nice refactor. This is fine, another option is disabling the delete button under the options menu with a tooltip on reason.

Reply:
I would be fine with that approach.  Keeps the logic out of the modal.  Do we have a design pattern for a disabled options menu option? @chudo76

Reply:
One option is disabling the delete button but showing it under the options menu.
This looks like scope creep, we can do a follow up maybe. I noticed we don't have tooltip options on a button today (unless they are table/top-bar buttons).

Reply:
Good point.  Will enter a Jira to address this later.

Reply:
> I would be fine with that approach. Keeps the logic out of the modal. Do we have a design pattern for a disabled options menu option? @chudo76

For Seed Bank Sub-locations, we just disable the button and have no info for why. For Projects we have a dialog box warning message (we don't prevent). At least with the dialog box we can let people know why they can't delete.

Dialog Box Title: "Delete Species"
Message: "This species is in use and cannot be deleted."
Button: "Got It"